extensions/amp-story/1.0/test/test-amp-story-grid-layer.js

```diff
@@ -126,4 +126,14 @@ describes.realWin('amp-story-grid-layer', {amp: true}, (env) => {
       )
     ).to.equal(562);
   });
+
+  it('should apply the preset attributes', async () => {
+    gridLayerEl.setAttribute('preset', '2021-foreground');
+    await buildGridLayer();
+
+    storeService.dispatch(Action.SET_PAGE_SIZE, {width: 1000, height: 1000});
+
+    expect(gridLayerEl.getAttribute('aspect-ratio')).to.equal('69:116');
+    expect(gridLayerEl).to.have.class('i-amphtml-story-grid-template-aspect');
```

Comment:
A unit test should only assert one thing:

- please make this test more specific, eg: "should add an aspect-ratio ratio attr from the preset"
- if you want to keep the second assert, create a new specific test
- please add an extra case with a wrong preset name

Reply:
Yes! I want to keep the second one which ensures the aspect-ratio attribute is being actually used (which could fail if it gets parsed before the preset is applied for instance).

I'll write those tests 👍